// api/mockProducts.js
export default [
    {
        id: '1',
        categoryId: '1',
        name: 'Orange',
        description:
            'An orange, also called sweet orange to distinguish it from the bitter orange (Citrus × aurantium), is the fruit of a tree in the family Rutaceae',
        price: 2,
        image: '/uploads/апельсин.jpg',
        weight: '1 kg',
        calories: '36/100grams'

    },
    {
        id: '2',
        categoryId: '1',
        name: 'banana',
        description:
            'A banana is an elongated, edible fruit – botanically a berry – produced by several kinds of large herbaceous flowering plants in the genus Musa',
        price: 1.5,
        image: '/uploads/банан.jpg',
        weight: '1 kg',
        calories: '89/100grams'
    },
    {
        id: '3',
        categoryId: '1',
        name: 'grapes',
        description:
            'Grapes are a type of fruit that grow in clusters of 15 to 300, and can be crimson, black, dark blue, yellow, green, orange, and pink. ',
        price: 3,
        image: '/uploads/виноград.jpg',
        weight: '1 kg',
        calories: '73/100grams'
    },
    {
        id: '4',
        categoryId: '2',
        name: 'Ice-Cream',
        description:
            'Ice cream is a dessert food product, which is a mass frozen during the process of continuous whipping.',
        price: 1.72,
        image: '/uploads/ice.jpg',
        weight: '330 grams',
        calories: '207/100grams'
    },
    {
        id: '5',
        categoryId: '2',
        name: 'Cake',
        description:
            'Sweet confectionery product made from butter dough with cream, "Chocolate cake"',
        price: 7.4,
        image: '/uploads/cake.jpg',
        weight: '1.5 kg',
        calories: '292/100grams'
    },
    {
        id: '6',
        categoryId: '3',
        name: 'Coca-Cola',
        description:
            'Coca-Cola is a carbonated soft drink produced by the Coca-Col company.',
        price: 1.2,
        image: '/uploads/coca.jpg',
        weight: '1 l',
        calories: '42/100grams'
    }
]
